net-utils.js

```diff
@@ -1,53 +1,94 @@
 //GET ALL URLS - DEFAULT BIN :)
 const axios = require("axios");
-// const koren = require("@korenezri/jsondb");
+const base_url = "http://localhost:3001";
 
-const readBin = async () => {
+const readBin = async (location) => {
+  if (process.env.NODE_ENV === "test") {
+    location = "test";
+  }
+  if (!location) location = "default";
```

Comment:
Good. There is a shortened version with ternary operator `const var = a ? b : c`